examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py

```diff
@@ -333,7 +340,7 @@ def eval_func(model):
         eval_func=eval_func,
     )
 
-    q_model.save(args.output_dir)
+    # q_model.save(args.output_dir)
```

Comment:
why remove save?

Reply:
remove for test. Now "save" has been restored.